playbook.md

```diff
@@ -1223,26 +1118,22 @@ But there are some limitations. Under the support service, we do not:
   Directory and suggest one, which we can then check and install.
 * Fix complex problems that come up after a plugin or WordPress core update is
   applied. Exactly what constitutes a complex problem is at our discretion. But
-  if we've spent a couple of hours on an update-related bug and it's still not fixed,
+  if we've spent a couple of hours on an updated-related bug and it's still not fixed,
```

Comment:
"update-related" is the correct choice here, we're using "update" as a noun (expanded, this would be "a WordPress update-related bug").